packages/devtools-launchpad/.babelrc

```diff
@@ -10,7 +10,7 @@
 
     ["module-resolver", {
       "alias": {
-        "react-dom": "react-dom/dist/react-dom",
+        "react-dom": "react-dom/umd/react-dom.development",
```

Comment:
Yeah, sorry. I wasn't really trying to answer your question so much as confirming that I understood what it was doing.

Your assertion seems legit.